Gems/Atom/RHI/DX12/Code/Source/RHI/Device.cpp

```diff
@@ -174,6 +174,8 @@ namespace AZ
             m_limits.m_minConstantBufferViewOffset = D3D12_CONSTANT_BUFFER_DATA_PLACEMENT_ALIGNMENT;
             m_limits.m_maxIndirectDrawCount = static_cast<uint32_t>(-1);
             m_limits.m_maxIndirectDispatchCount = static_cast<uint32_t>(-1);
+            m_limits.m_maxConstantBufferSize = static_cast<uint32_t>(-1);
+            m_limits.m_maxBufferSize = static_cast<uint32_t>(-1);
```

Comment:
I couldn't find what were the limits for DX12, so I left them at the maximum value. If somebody knows this please let a comment below.